es-topic-indexer/example/indexer-params.yaml

```diff
@@ -0,0 +1,30 @@
+apiVersion: apps/v1
+kind: Deployment
+metadata:
+  name: &app es-topic-indexer
+spec:
+  template:
+    spec:
+      containers:
+        - name: *app
+          env:
+            - name: ES_HOSTS
```

Comment:
why do repeat these env vars here and in `es-topic-indexer.yaml`?

Reply:
I see your point, and I get that the default values might be "dangerous".  I was split if I should leave them here or not, but decided to keep it.
The other manifests specified in this repository also have some default values. I wanted to create a base that is a complete example on how the rendered, minimal resource should look like.
Directory "example" has another role- it shows the most standard modifications to the manifest.

Reply:
> I guess we could link to our running instance in dev-enablment once that's going (is this what we did with other services, I can't remember)

We didn't linked our services here, even if they are almost a carbon copy of the example. 
It is so that anyone changing the manifest can change the example (and we hope for the teams contributing here :slightly_smiling_face: ), also- with this approach we don't need to bother if our services have special needs (e.g.- our example of es-topic-indexer communicates with Kafka in another cluster, so it needs special annotation). 

The con of this is that we are not running the example anywhere, just as you said. 
Only thing coming to my mind to help this state is, to add workflow similar as in kubernetes-manifests here, showing the change in rendererd "example".